pkg/controllers/nodeclaim/garbagecollection/controller.go

```diff
@@ -101,8 +101,8 @@ func (c *Controller) Reconcile(ctx context.Context) (reconcile.Result, error) {
 		}
 		log.FromContext(ctx).WithValues(
 			"NodeClaim", klog.KRef("", nodeClaims[i].Name),
+			"Node", klog.KRef("", nodeClaims[i].Status.NodeName),
 			"provider-id", nodeClaims[i].Status.ProviderID,
-			"nodepool", nodeClaims[i].Labels[v1.NodePoolLabelKey],
```

Comment:
Would removing this affect troubleshooting time?

Reply:
I don't think so -- NodeClaims use generate-name so you can search for the nodepool as a prefix and get all nodeclaims associated with i